VisualizationBase/src/Scene.cpp

```diff
@@ -109,11 +112,24 @@ ModelRenderer* Scene::defaultRenderer()
 	return &defaultRenderer_;
 }
 
-void Scene::addTopLevelItem(Item* item)
+void Scene::addTopLevelNode(Node *node, int column, int row)
+{
+	currentViewItem()->insertNode(node, column, row);
+}
+
+void Scene::addTopLevelItem(Item* item, bool show)
 {
 	Q_ASSERT(!inAnUpdate_);
+	//If we have a ViewItem, we must add it to all ViewItems
+	if (auto view = dynamic_cast<ViewItem*>(item))
+	{
+		if (viewItems_.size() == 0)
```

Comment:
As we discussed before, please remove this code. A view should already be added to the viewItems by some other dedicated method.